cmd/util/cmd/execution-data-blobstore/cmd/root.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ipfs/go-blockservice"
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	badger "github.com/ipfs/go-ds-badger2"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/onflow/flow-go/module/blobs"
+	"github.com/onflow/flow-go/network"
+	"github.com/onflow/flow-go/network/mocknetwork"
+)
+
+var (
+	flagBlobstoreDir string
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "execution-data-blobstore",
+	Short: "interact with execution data blobstore",
+}
+
+var RootCmd = rootCmd
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println("error", err)
+		os.Exit(1)
+	}
+}
+
+func initBlobservice() (network.BlobService, datastore.Batching) {
+	ds, err := badger.NewDatastore(flagBlobstoreDir, &badger.DefaultOptions)
+
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not init badger datastore")
+	}
+
+	bstore := blockstore.NewBlockstore(ds)
+
+	blockService := blockservice.New(bstore, nil)
+
+	bs := new(mocknetwork.BlobService)
+
+	bs.
```

Comment:
nit: This looks super out of place, can we just follow the regular convention of having the first func on the same line?

nit^2: all three things here are "bs", could we just spell this one out too? `blockService`